test_conformance/pipes/test_pipe_info.cpp

```diff
@@ -52,29 +52,37 @@ int test_pipe_info( cl_device_id deviceID, cl_context context, cl_command_queue
 
     if(pipe_depth != returnVal)
     {
-        log_error( "Error in clGetPipeInfo() check of pipe max packets\n" );
-        return -1;
+        test_fail("Error in clGetPipeInfo() check of pipe max packets\n");
     }
     else
     {
         log_info( " CL_PIPE_MAX_PACKETS passed.\n" );
     }
 
-    err = create_single_kernel_helper_with_build_options(context, &program, &kernel, 1, (const char**)&pipe_kernel_code, "pipe_kernel", "-cl-std=CL2.0 -cl-kernel-arg-info");
-    test_error_ret(err, " Error creating program", -1);
+    err = create_single_kernel_helper_with_build_options(
+        context, &program, &kernel, 1, &pipe_kernel_code, "pipe_kernel",
+        "-cl-std=CL2.0 -cl-kernel-arg-info");
```

Comment:
This isn't an issue you introduced, but if you feel up to fixing it: hard-coding `-cl-std=CL2.0` here will cause a problem for an OpenCL 3.0 implementation that supports pipes but not some other feature that is required for OpenCL C 2.0.  It'd be better to use a solution like `get_build_options()` in test_kernel_arg_info.cpp.

Reply:
A good suggestion, but I think that would go beyond the intended scope of these changes.